src/main/java/io/cdap/plugin/salesforce/SalesforceSchemaUtil.java

```diff
@@ -53,6 +53,7 @@
     .put(FieldType.datetime, Schema.of(Schema.LogicalType.TIMESTAMP_MICROS))
     .put(FieldType.time, Schema.of(Schema.LogicalType.TIME_MICROS))
     .put(FieldType.string, Schema.of(Schema.Type.STRING))
+    .put(FieldType.complexvalue, Schema.arrayOf(Schema.of(Schema.Type.STRING)))
```

Comment:
Does this also support array of types other than string?

Reply:
Yes, it is also supporting arrays or other types, that implementation is written in getSchemaWithFields method.